<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MyClientWindow</class>
 <widget class="QWidget" name="MyClientWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1200</width>
    <height>800</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QTextEdit" name="SendText">
   <property name="geometry">
    <rect>
     <x>800</x>
     <y>675</y>
     <width>350</width>
     <height>60</height>
    </rect>
   </property>
  </widget>
  <widget class="QTextEdit" name="MessageText">
   <property name="geometry">
    <rect>
     <x>800</x>
     <y>500</y>
     <width>350</width>
     <height>150</height>
    </rect>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="Send">
   <property name="geometry">
    <rect>
     <x>900</x>
     <y>740</y>
     <width>150</width>
     <height>50</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QTextEdit" name="HostIPText">
   <property name="geometry">
    <rect>
     <x>890</x>
     <y>370</y>
     <width>261</width>
     <height>41</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="ConnectToHost">
   <property name="geometry">
    <rect>
     <x>800</x>
     <y>420</y>
     <width>190</width>
     <height>60</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="ESC">
   <property name="geometry">
    <rect>
     <x>1140</x>
     <y>20</y>
     <width>40</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="HostIP">
   <property name="geometry">
    <rect>
     <x>800</x>
     <y>370</y>
     <width>81</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:18pt;&quot;&gt;HostIP&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="NameText">
   <property name="geometry">
    <rect>
     <x>900</x>
     <y>60</y>
     <width>181</width>
     <height>61</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>800</x>
     <y>70</y>
     <width>91</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:18pt;&quot;&gt;Gamer&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Ready">
   <property name="geometry">
    <rect>
     <x>1030</x>
     <y>420</y>
     <width>120</width>
     <height>60</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
